tests/components/caldav/test_calendar.py

```diff
@@ -1085,10 +1086,11 @@ async def test_calendar_components(hass: HomeAssistant) -> None:
 @freeze_time(_local_datetime(17, 30))
 async def test_setup_config_entry(
     hass: HomeAssistant,
-    setup_integration: Callable[[], Awaitable[bool]],
+    config_entry: MockConfigEntry,
 ) -> None:
     """Test a calendar entity from a config entry."""
-    assert await setup_integration()
+    config_entry.add_to_hass(hass)
+    await config_entry.async_setup(hass)
```

Comment:
Please always use `hass.config_entries.async_setup` to set up the config entry.

Reply:
Yes, of course,  I think there are a few instances of this i added to a few integration tests and i'll take a pass and clean them up.